full/atm_land_ice_flux_exchange.F90

```diff
@@ -212,6 +220,9 @@ module atm_land_ice_flux_exchange_mod
        ex_e_tr_n,     & !< coefficient in implicit scheme
        ex_f_tr_delt_n   !< coefficient in implicit scheme
 
+  !deposition velocity at reference height and atmospheric height
+  real, allocatable, dimension(:,:) :: ex_tr_con_ref, ex_tr_con_atm
```

Comment:
Please switch the documentation to doxygen format.  The variables below show an example of the format using `!<` for each variable.